src/chapter01/exercises/exercise02.lean

```diff
@@ -9,13 +10,74 @@ def E : set ℝ := {1}
 -- Which pair of these sets has the property that neither is contained in the other?
 lemma parta : ∃ S T ∈ ({B, C, D, E} : set (set ℝ)), ¬ (S ⊆ T) ∧ ¬ (T ⊆ S) :=
 begin
-  sorry
+  have h0 : D = ({0} : set ℝ),
+  { simp_rw D,
+    ext,
+    split,
+    { intro hx,
+      simp at hx,
```

Comment:
You don't need to do this `simp` call, it doesn't actually do much. Use `squeeze_simp` to see what it's doing, but it's just changing x^2<1 to |x|<1 and then rearranging stuff. I would put this off until later.